org.cvfosammmm.Porto.json

```diff
@@ -1,376 +1,73 @@
-                    "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/be/39/32da3184734730c0e4d3fa3b2b5872104668ad6dc1b5a73d8e477e5fe967/Pygments-2.5.2-py2.py3-none-any.whl",
-                    "sha256": "2a3fe295e54a20164a9df49c75fa58526d3be48e14aceba6d6b1e8ac0bfd6f1b"
-                }
-            ]
+            "cleanup": [ "/lib/cmake" ]
         },
+        "python3-requirements.json",
         {
-            "name": "python3-PyGObject",
-            "buildsystem": "simple",
-            "build-commands": [
-                "source /app/bin/activate",
-                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} PyGObject"
-            ],
+            "name": "gtksourceview",
+            "config-opts": ["--enable-gtk-doc=no"],
             "sources": [
                 {
-                    "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/46/8a/b183f3edc812d4d28c8b671a922b5bc2863be5d38c56b3ad9155815e78dd/PyGObject-3.34.0.tar.gz",
-                    "sha256": "2acb0daf2b3a23a90f52066cc23d1053339fee2f5f7f4275f8baa3704ae0c543"
-                },
-                {
-                    "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/25/95/ac361f06789c7dad19c37f4d7043df9275f416dffcdc44e22c3befeb7b08/pycairo-1.19.0.tar.gz",
-                    "sha256": "4f5ba9374a46c98729dd3727d993f5e17ed0286fd6738ed464fe4efa0612d19c"
-                },
-                {
-                    "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/8c/23/848298cccf8e40f5bbb59009b32848a4c38f4e7f3364297ab3c3e2e2cd14/wheel-0.34.2-py2.py3-none-any.whl",
-                    "sha256": "df277cb51e61359aba502208d680f90c0493adec6f0e848af94948778aed386e"
-                },
-                {
-                    "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/3d/72/1c1498c1e908e0562b1e1cd30012580baa7d33b5b0ffdbeb5fde2462cc71/setuptools-45.2.0-py3-none-any.whl",
-                    "sha256": "316484eebff54cc18f322dea09ed031b7e3eb00811b19dcedb09bc09bba7d93d"
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz",
+                    "sha256": "691b074a37b2a307f7f48edc5b8c7afa7301709be56378ccf9cc9735909077fd"
                 }
             ]
         },
         {
             "name": "porto",
             "buildsystem": "simple",
             "build-commands": [
-                "source /app/bin/activate",
-                "mkdir -p /app/share/applications",
-                "cp resources/org.cvfosammmm.Porto.desktop /app/share/applications/org.cvfosammmm.Porto.desktop",
+                "mkdir -p ${FLATPAK_DEST}/share/applications",
+                "cp resources/org.cvfosammmm.Porto.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop",
```

Comment:
```suggestion
                "install -Dm644 resources/org.cvfosammmm.Porto.desktop  ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop",
```

Reply:
annd similar below but I can't suggest because github.

tl;dr install does create the directories and is better than `cp` to install files.

Reply:
Thanks!